pywwt/core.py

```diff
@@ -117,16 +138,34 @@ def center_on_coordinates(self, coord, fov=60*u.deg, instant=True):
                        instant=instant)
 
     def set_current_time(self, dt):
-        self._send_msg(event='set_datetime',
-                       year=dt.year, month=dt.month, day=dt.day,
-                       hour=dt.hour, minute=dt.minute, second=dt.second,
-                       millisecond=int(dt.microsecond / 1000.))
+        """
+        Set the viewer time to match the real-world time.
+
+        Parameters
+        ----------
+        dt : `~datetime.datetime` or `~astropy.time.core.Time`
+            The current time, either as a `datetime` object or an astropy
+            `Time`.
+        """
+        if isinstance(dt, t.core.Time):
+            dt = dt.datetime
+            self._send_msg(event='set_datetime',
```

Comment:
Just un-indent the self._send_msg and then remove the elif statement below. That is, you can just do:

```
if isinstance(dt, t.core.Time):
    dt = dt.datetime
self._send_msg(...)
```